qiskit/circuit/commutation_checker.py

```diff
@@ -25,6 +25,27 @@
 _skipped_op_names = {"measure", "reset", "delay", "initialize"}
 _no_cache_op_names = {"annotated"}
 
```

Comment:
Cannot comment on the right line but it looks like the file is missing the `from __future__ import annotations`

Reply:
Thanks! I updated to just using the `typing` module like the rest of the code in 351d5f1, since it'll be ported to rust anyways 🙂